data/quests_strings/noquest.csv

```diff
@@ -27,42 +27,37 @@ NOQUEST_POSITIVE,👍,,,,
 NOQUEST_NEGATIVE,👎,,,,
 NOQUEST_NAV_FWD,❯,,,,
 NOQUEST_NAV_BAK,❮,,,,
-NOQUEST_NOTINSTALLED,"Uh-oh! It looks like this App isn't installed!
-
-You need to download it from the App Center if you want to play.
-
-Click the icon named <tt><span insert_hyphens=""false"" foreground=""#287A8C"" background=""#FFFFFF"">App Center</span></tt> on the Desktop, then search for the program you want to install.",daemon,,,
-NOQUEST_NOCONNECTION,"Uh-oh! It looks like you don't have an active internet connection.
-
-We need an active internet connection to display the steps and materials for this activity.
+NOQUEST_NOTINSTALLED_GENERIC,Uh-oh! It looks like the App this quest needs isn't installed! Would you like me to help you install it?,daemon,,,
+NOQUEST_AUTOINSTALL,"OK, I'll try to start the App install, this shouldn't take too long!",daemon,,,

```

Comment:
It is a bit unclear what happens here. In EOS GNOME Software is started (on the detail page) but the dialogue suggests that the app might be installed automatically without further intervention. But the user still has to start the installation.